python/dlisio/ext/lis.cpp

```diff
@@ -365,9 +365,10 @@ void init_lis_extension(py::module_ &m) {
     ;
 
     py::class_< lis::record_index >( m, "lis_record_index" )
-        .def( "explicits", &lis::record_index::explicits )
-        .def( "implicits", &lis::record_index::implicits )
-        .def( "size",      &lis::record_index::size )
+        .def( "explicits",    &lis::record_index::explicits )
+        .def( "implicits",    &lis::record_index::implicits )
+        .def( "size",         &lis::record_index::size )
+        .def( "isincomplete", &lis::record_index::is_incomplete)
```

Comment:
why isincomplete and not is_incomplete?

Reply:
Convension. In c++ it's common to use `is_`. e.g. [1], while in python it's common to to drop the underscore. e.g. [2]
[1] https://en.cppreference.com/w/cpp/types/is_same
[2] https://docs.python.org/3/library/functions.html#isinstance